src/core/transient_variable.jl

```diff
@@ -19,15 +19,15 @@ function variable_density(
     end
 
     report &&
-        _IM.sol_component_value(gm, nw, :junction, :density, ids(gm, nw, :junction), rho)
+        _IM.sol_component_value(gm, gm_it_sym, nw, :junction, :density, ids(gm, nw, :junction), rho)
     report &&
-        _IM.sol_component_value(gm, nw, :junction, :pressure, ids(gm, nw, :junction), rho)
+        _IM.sol_component_value(gm, gm_it_sym, nw, :junction, :pressure, ids(gm, nw, :junction), rho)
```

Comment:
This is a stylistic point, but in PowerModels we implemented a package specific version of `sol_component_value` that is specialized on `AbstractPowerModel`, so that `pm_it_sym` does not need to be used explicitly within the package.

https://github.com/lanl-ansi/PowerModels.jl/blob/rc-v0.18/src/core/solution.jl#L28